explorer/puller/model/Block.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime, timedelta
+
+from symbolchain.CryptoTypes import PublicKey
+
+
+class Block:
+	NEM_EPOCH = datetime(2015, 3, 29, 0, 6, 25, 0)
+
+	def __init__(self, height, timestamp, total_fees, total_transactions, difficulty, block_hash, signer):
+		"""Create Block model."""
+
+		# pylint: disable=too-many-arguments
+
+		self.height = height
+		self.timestamp = timestamp
+		self.total_fees = total_fees
+		self.total_transactions = total_transactions
+		self.difficulty = difficulty
+		self.block_hash = block_hash
+		self.signer = signer
+
+	@staticmethod
+	def convert_timestamp_to_datetime(timestamp):
+		return (Block.NEM_EPOCH + timedelta(seconds=timestamp)).strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
this should happen via facade:
`facade.network.datetime_converter.to_datetime(timestamp)`